modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/dns_failure_test/DnsBlocker.java

```diff
@@ -14,46 +14,96 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package java.net;
+package org.apache.ignite.internal.ducktest.tests.dns_failure_test;
 
 import java.io.File;
 import java.io.FileNotFoundException;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
 import java.text.SimpleDateFormat;
 import java.util.Arrays;
 import java.util.Date;
 import java.util.Scanner;
+import org.apache.ignite.startup.cmdline.CommandLineStartup;
 
 /** */
-public class DnsBlocker {
+public class DnsBlocker implements NameServiceHandler {
```

Comment:
Now functionality of the class not only to block DNS requests, but to start Ignite with blocked DNS, so class name should be renamed.